libbeat/scripts/generate_fields_docs.py

```diff
@@ -70,6 +84,14 @@ def document_field(output, field, field_path):
     if "path" in field:
         output.write("alias to: {}\n\n".format(field["path"]))
 
+    # For Apm-Server docs only
+    # Assign an ECS badge for fields containing "overwrite"
+    if beat_title == "Apm-Server":
+        if "overwrite" in field:
+            # And it's not a Kubernetes field
+            if re.match("^kubernetes.*", field["field_path"]) is None:
```

Comment:
`kubernetes.*` fields had to be excluded here as they're not ECS fields, but APM marks them as `overwrite:true`.